# request
bool use_joint_values
---
# response
bool success